docs/cookbook/limit-repeated-requests.md

```diff
@@ -18,24 +18,58 @@ import * as rateLimit from 'express-rate-limit';
 import { AppController } from './app/app.controller';
 
 async function main() {
-    // Connection to the database(s)...
+  // Connection to the database(s)...
     
-    const expressApp = express();
-    expressApp.use(rateLimit({
-      max: 100, // limit each IP to 100 requests per windowMs
-      windowMs: 15 * 60 * 1000, // 15 minutes
-    }));
+  const expressApp = express();
+  expressApp.use(rateLimit({
+    max: 100, // limit each IP to 100 requests per windowMs
+    windowMs: 15 * 60 * 1000, // 15 minutes
+  }));
     
-    const app = createApp(AppController, expressApp); // For v1
-    // For v0.8
-    // const app = createApp(AppController, { /* ... */ }, expressApp);
+  const app = createApp(AppController, expressApp); // For v1
+  // For v0.8
+  // const app = createApp(AppController, { /* ... */ }, expressApp);
     
-    const httpServer = http.createServer(app);
-    const port = Config.get('port', 3001);
-    httpServer.listen(port, () => {
-        console.log(`Listening on port ${port}...`);
-    });
+  const httpServer = http.createServer(app);
+  const port = Config.get('port', 3001);
+  httpServer.listen(port, () => {
+    console.log(`Listening on port ${port}...`);
+  });
 }
 
 main();
 ```
+
+
+**Rate limiting with CORS**
+
+If you need CORS headers to be send back in a response, you will need to manually set the headers on the rate limiter response because it does not get handled by FoalTS hooks. If you don't manually set any headers only the default Express.js headers will be set in the response.
+
+> Note: Because the rate limiter response does not get handled by FoalTS, you should also set the default FoalTS headers manually.
+
+```typescript
+expressApp.use(rateLimit({
+  max: 100,
+  windowMs: 15 * 60 * 1000,
+  handler: function (req, res, next) {
+    // Set default FoalTS headers
+    res.removeHeader('X-Powered-By');
```

Comment:
The default headers can even be moved to the first code block as they are not specific to CORS.